tests/publiccloud/create_aistack_env.pm

```diff
@@ -181,13 +226,21 @@ sub test_openwebui_service {
     set_var('OPENWEBUI_IP', "$ipaddr");
     record_info("Added $ipaddr to /etc/hosts with hostname $host_name");
 
-    # connect open-webui service
-    my $curl_cmd = "curl -v -k https://$host_name";
-    my $curl_result = script_run($curl_cmd);
-    if ($curl_result == 0) {
-        record_info("Successfully connected to the open-webui service at $curl_cmd \n");
+    # get endpoints
+    my $endpoint_cmd = "kubectl get endpoints $sr_name -n $namespace -o=jsonpath='{.subsets[*].addresses[*].ip}'";
+    my $endpoint_result = script_run($endpoint_cmd);
```

Comment:
Given your reply in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20723/files#r1884169392, then here still needed to do:
```suggestion
    my $endpoint_result = script_output($endpoint_cmd);
```

Reply:
Updated the script..Here is the VR https://openqa.suse.de/tests/16204784#